src/xref2/tools.ml

```diff
@@ -1179,7 +1189,8 @@ and fragmap :
               if handled' then
                 map_include_decl i.decl sub >>= fun decl ->
                 let expansion_ =
-                  Component.Signature.{ items = items'; removed = removed' }
+                  Component.Signature.
+                    { items = items'; removed = removed'; compiled = false }
```

Comment:
Should it be `expansion_.compiled` ?

Reply:
no, same reason as the other one in fragmap.